lib/devices/core.js

```diff
@@ -14,7 +14,7 @@ module.exports = function core(device) {
    * @return {void}
    */
   device.ping = function(callback) {
-    return command(commands.ping, null, callback);
+    command(commands.ping, null, callback);
```

Comment:
Since command never returns did not make sense to me ATM. We can add later when needed is a pretty easy/quick upd.